doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -349,23 +348,23 @@ than naïve Bayes). We can change the learner by just plugging a different
 classifier object into our pipeline::
 
   >>> from sklearn.linear_model import SGDClassifier
-  >>> text_clf = Pipeline([('vect', CountVectorizer()),
+  >>> svm_pipe = Pipeline([('vect', CountVectorizer()),
   ...                      ('tfidf', TfidfTransformer()),
   ...                      ('clf', SGDClassifier(loss='hinge', penalty='l2',
   ...                                            alpha=1e-3, random_state=42,
   ...                                            max_iter=5, tol=None)),
   ... ])
-  >>> text_clf.fit(twenty_train.data, twenty_train.target)  # doctest: +ELLIPSIS
+  >>> svm_pipe.fit(twenty_train.data, twenty_train.target)  # doctest: +ELLIPSIS
   Pipeline(...)
-  >>> predicted = text_clf.predict(docs_test)
-  >>> np.mean(predicted == twenty_test.target)            # doctest: +ELLIPSIS
+  >>> svm_pred = svm_pipe.predict(docs_test)
+  >>> np.mean(svm_pred == twenty_test.target)            # doctest: +ELLIPSIS
   0.912...
 
 ``scikit-learn`` further provides utilities for more detailed performance
-analysis of the results::
+analysis of the results. Two useful functions are :func:`sklearn.metrics.classification_report` and :func:`sklearn.metrics.confusion_matrix`, which are as follows::
```

Comment:
Ahh okay that's fine, I wouldn't be offended at all if you didn't merge this PR. I was just following the tutorial and found places where I would do things differently which made more sense to me.